### ISSUE TITLE
FlatHub

### ISSUE THREAD

**Issue by Eric1212** [UnnoTed/wireguird#48]
Great Software, simple, work, it's would be nice of it's be available on FlatHub for keeping it on date and maybe, providing more interest to this project !

Thanks,
Eric.

**Comment by rugk** [comment]
I agree, it would be nice if you could publish this as a [flatpak](https://flatpak.org/) on [flathub](https://flathub.org/) e.g.

Flatpaks are a new software distribution mechanism for Linux distros, can thus installed on any distro and are easy to update. They are easy to install _and_ update and work on all Linux distros.
Also, if you publish it on _FlatHub_ you may grow your user base given many distros include that and it is a common software source, so your app can be discovered more easily. 

Here is [how to get started](http://docs.flatpak.org/en/latest/getting-started.html).

The thing could be managing VPN inside a sandbox could be tricky. That said, of it just manages a server that is installed on the host, it is very easy.

**Comment by aaasoft** [comment]
Agree.wish to see this app on flathub.

**Comment by mbnoimi** [comment]
+1000000
